### ISSUE TITLE
networkInterfaces properties doesn't allow for multiple digits.

### ISSUE THREAD

**Issue by ad0pl** [F5Networks/f5-telemetry-streaming#18]
##### ISSUE TYPE

 - Bug Report

##### Telemetry Streaming 1/ 1.4.0

```

```

##### BIGIP VERSION

```
Sys::Version
Main Package
  Product     BIG-IP
  Version     13.1.1.3
  Build       0.0.1
  Edition     Point Release 3
  Date        Wed Nov 28 18:50:45 PST 2018
```

##### SUMMARY

The renameKeys property on networkInterfaces (on or around line 166) regex pattern doesn't allow for more than 1 digit characters in the interface identifier.  That is interfaces 1.10 - 1.19 will overwrite the properties for interface 1.1; interfaces 1.20 - 1.29 will over write the properties for interface 1.2 and so on.  Assumingly from the current regex if there were an interface 10.1, that too would over write stats for interface 1.1
```

```

##### STEPS TO REPRODUCE

```

```

##### EXPECTED RESULTS

Expected interface 1.1 - 1.24 to be listed in telemetry sent to my consumer and interface status of interface 1.1 and 1.2 to be "up".
##### ACTUAL RESULTS

Only interfaces 1.0 (doesn't even exist) - 1.9 were sent and interface status of 1.1 and 1.2 reflective of the status of interface 1.19 and 1.24 respectively.

```
{"aggr_period":300,"device_base_mac":"00:0a:49:c1:e1:40","devicegroup":"","facility":"","interface_name":"1.1","interface_status":"uninit"}
```

**Comment by ad0pl** [comment]
Assumingely the below patch would work.

--- properties.json-orig	2019-07-25 15:32:17.000000000 -0500
+++ properties.json	2019-07-25 15:32:39.000000000 -0500
@@ -163,7 +163,7 @@
             "structure": { "parentKey": "system" },
             "key": "networkInterfaces",
             "filterKeys": { "exclude": [ "counters.errorsAll", "counters.dropsAll", "counters.pktsIn", "counters.pktsOut", "mediaActive", "tmName" ] },
-            "renameKeys": { "patterns": { "net/interface": { "pattern": "([0-9]\\.[0-9]|(?!\/)mgmt(?=\/))" } } }
+            "renameKeys": { "patterns": { "net/interface": { "pattern": "([0-9]+\\.[0-9]+|(?!\/)mgmt(?=\/))" } } }
         },
         "provisioning": {
             "structure": { "parentKey": "system" },

**Comment by petrov-serg** [comment]
Thank you, @ad0pl 

Internal issue created, ID - AUTOTOOL-557

**Comment by dstokesf5** [comment]
This issue has been resolved as part of the 1.12.0 release.
